osu.Game/Storyboards/Drawables/DrawableStoryboard.cs

```diff
@@ -81,7 +81,21 @@ private void load(FileStore fileStore, GameplayClock clock, CancellationToken? c
             if (clock != null)
                 Clock = clock;
 
-            dependencies.Cache(new TextureStore(host.CreateTextureLoaderStore(fileStore.Store), false, scaleAdjust: 1));
+            TextureStore textureStore = new TextureStore(host.CreateTextureLoaderStore(fileStore.Store), false, scaleAdjust: 1);
+
+            dependencies.Cache(textureStore);
+
+            double storyboardLatestEventTime = Storyboard.LatestEventTime ?? 0;
+
+            var drawableBackground = Storyboard.CreateSpriteFromResourcePath(Storyboard.BeatmapInfo.Metadata?.BackgroundFile, textureStore);
+
+            if (drawableBackground != null && !Storyboard.ReplacesBackground && storyboardLatestEventTime != 0)
+            {
+                var sprite = new StoryboardSprite(Storyboard.BeatmapInfo.Metadata?.BackgroundFile, Anchor.Centre, new Vector2(320, 240));
+                sprite.TimelineGroup.Scale.Add(Easing.None, 0, 0, 1, 480f / drawableBackground.Height);
+                sprite.TimelineGroup.Alpha.Add(Easing.None, 0, storyboardLatestEventTime, 1, 1);
```

Comment:
As an alternative to all of this manual poking and prodding, maybe there should be a new `IStoryboardElement` implementation *just* for the background, rather than trying to reuse `StoryboardSprite`? Then you could `CreateDrawable()` and create/load the background once (not twice, as done above), and also set the height to 480 once without having to calculate scale.